### ISSUE TITLE
[Priority: Low] Show multiple receive addresses

### ISSUE THREAD

**Issue by nopara73** [stratisproject/Breeze#203]
A user that often gets paid wants to create new addresses for every payment, right now the receive address only changes if money is sent to it.  
I propose either  
1. Shuffle a couple of receive addresses and randomly show them up,
2. Show multiple receive addresses at once.

**Comment by dev0tion** [comment]
When we started Breeze we wanted to include an "advanced" switch in the settings, making this like that possible.
What do you think @bokobza?

**Comment by bokobza** [comment]
yeah I think ultimately we'd like something like that. Let's leave this open and assign it to vNext :-)

**Comment by nopara73** [comment]
The API is ready to do this feature as soon as PR https://github.com/stratisproject/StratisBitcoinFullNode/pull/506 merged.  
Endpoint: `addresses`
